File: B/workdir/Task0.Samples/BC++3.1/Task0.cpp
#include <fstream.h>


const char
  *inputFileName = "input.txt",
  *outputFileName = "output.txt";


void main() {
  ifstream input(inputFileName);
  ofstream output(outputFileName);

  int a, b;
  input >> a >> b;

  int c = a + b;

  output << c << endl;

  input.close();
  output.close();
}

File: B/workdir/Task0.Samples/BP7.0/Task0.pas
program Task0;

const
  InputFileName = 'input.txt';
  OutputFileName = 'output.txt';

var
  A, B, C: Integer;
  Input, Output: Text;

begin
  Assign(Input, InputFileName);
  Reset(Input);
  Assign(Output, OutputFileName);
  Rewrite(Output);

  ReadLn(Input, A, B);

  C := A + B;

  WriteLn(Output, C);

  Close(Input);
  Close(Output);
end.
